lib/Backends/Interpreter/InterpreterNodes.cpp

```diff
@@ -395,7 +395,6 @@ void BoundInterpreterFunction::fwdConvTransposeInstFloatImpl(
 
   assert(idim.c % group == 0 && "Input channels must be divisible by group.");
   assert(odim.c % group == 0 && "Output channels must be divisible by group.");
-  assert(group == 1 && "Group must be 1.");
```

Comment:
Why did we have this here, if no other changes were necessary to support group > 1? @vuzelac-cadence

Reply:
This was improper guarding against an untested feature; 
This assert should have been in Nodes.cpp, before even trying to use group in calculations.

Reply:
This assert is there at LLVMIRGen.cpp. I have added the operator test and ONNXImporter test for the group. @vuzelac-cadence can you please check and comment on the changes in verify function.